vmdb/lib/workers/event_catcher_kubernetes.rb

```diff
@@ -5,7 +5,7 @@ def event_monitor_handle
     require 'kubernetes/events/kubernetes_event_monitor'
     @event_monitor_handle ||= KubernetesEventMonitor.new(
       @ems.api_endpoint,
-      @ems.api_version
+      EmsKubernetes.kubernetes_version
```

Comment:
I see.  Even so, the column _should_ have a value, because it's reportable, and then the report creator can know the version.  I guess the problem comes if you want to change it, but you want to avoid data migrations?  If that's the case you can override the column accessor with a normal ruby method that returns the constant, and also make the setter private.  Then you can still use the method .api_version.